lua/browsher/init.lua

```diff
@@ -17,15 +17,8 @@ local function get_open_command()
             return config.options.open_cmd
         end
     end
-    if vim.fn.has("unix") == 1 then
-        return { "xdg-open" }
-    elseif vim.fn.has("macunix") == 1 then
-        return { "open" }
-    elseif vim.fn.has("win32") == 1 then
-        return { "explorer.exe" }
-    else
-        return nil
-    end
```

Comment:
I kind of like the automatic detection for the open command. I personally like verifying the link looks/opens correct.

I like the clipboard idea though. 
Instead do you think it makes sense to have an early exit above the OS detection for `"clipboard"`?